### PR TITLE
Fix inconsistent coding convention

### PR DESCRIPTION
**Pull request by no1wudi**

### CODE REVIEW

File: core/iwasm/common/wasm_shared_memory.c [hunk 1]
```diff
@@ -232,14 +232,14 @@ destroy_wait_info(void *wait_info)
 }
 
 static void
-map_try_release_wait_info(HashMap *wait_map_, AtomicWaitInfo *wait_info,
+map_try_release_wait_info(HashMap *wait_map, AtomicWaitInfo *wait_info,
```
Comment (wenyongh) [inline core/iwasm/common/wasm_shared_memory.c:235]: This argument may shadow the variable defined in L44 and cause compilation warning, had better use other name here.

Comment (wenyongh) [reply core/iwasm/common/wasm_shared_memory.c:235]: Or how about removing this argument, as this function is only called by wasm_runtime_atomic_wait, which always passes the global wait_map to this function.

Comment (no1wudi) [reply core/iwasm/common/wasm_shared_memory.c]: Fixed, but it's strange, my compiler don't report the shadowing

**Review by wenyongh — APPROVED** [review]

File: core/iwasm/common/wasm_shared_memory.c (hunk 1 above)
Comment (wenyongh) [reply core/iwasm/common/wasm_shared_memory.c]: Yes, maybe it is only reported by some compiler.

**Pull request merged**